crates/eth-rpc/tests/rpc.rs

```diff
@@ -1,20 +1,93 @@
 #![recursion_limit = "1024"]
+mod kkrt_prep;
 mod utils;
 
 #[cfg(test)]
 mod tests {
     use std::str::FromStr;
 
+    use dojo_test_utils::sequencer::TestSequencer;
     use kakarot_rpc::eth_api::EthApiServer;
+    use kakarot_rpc::eth_rpc::KakarotEthRpc;
+    use kakarot_rpc_core::client::config::StarknetConfig;
+    use kakarot_rpc_core::client::KakarotClient;
     use kakarot_rpc_core::mock::assert_helpers::{assert_block, assert_block_header, assert_transaction};
-    use reth_primitives::{BlockNumberOrTag, H160, H256, U256, U64};
+    use reth_primitives::hex_literal::hex;
+    use reth_primitives::{Address, BlockNumberOrTag, H160, H256, U256, U64};
     use reth_rpc_types::Index;
     use serde_json::json;
     use starknet::core::types::{FieldElement, Transaction as StarknetTransaction};
     use starknet::macros::felt;
 
+    use crate::kkrt_prep::{create_call_request, create_raw_tx, init_kkrt_state};
     use crate::utils::setup_kakarot_eth_rpc;
 
+    #[tokio::test]
+    #[ignore]
+    async fn use_test_sequencer_test() {
+        let sn_test_sequencer = TestSequencer::start().await;
+        // TODO: lift from env
```

Comment:
having an env file for these tests that has 

- the evm private key 
- local kakarot repo 
  - can take an optional commit value 
  - script can build kkrt at a particular commit and move it to where the test system needs it